### PR TITLE
CSS/Material updates, design account profile form

### PR DESCRIPTION
**Pull request by emcapone**
I went through all the pages I have now and removed elements like hr/br and made some other minor style changes. I also added a dialog for confirming actions like deleting a pet or cancelling out of a form, and I designed the account profile form.

### CODE REVIEW

File: src/app/account-profile/account-profile.component.html [hunk 1]
```diff
@@ -1 +1,43 @@
-<p>Placeholder for account profile</p>
+<h1>Account Settings</h1>
+<form>
```
Comment (swhook52) [inline src/app/account-profile/account-profile.component.html:2]: You should look at [Reactive Forms](https://angular.io/guide/reactive-forms) which have you manage a form group where you can control required fields and validation in TS. It makes for a cleaner HTML form and much more declarative in TS.

The typical flow of a form is:
```html
<form [formGroup]="formGroup">
  <mat-form-field appearance="standard">
    <mat-label>Example Field</mat-label>
    <input matInput formControlName="exampleField">

    <mat-spinner matSuffix *ngIf="isLoading" diameter="20"></mat-spinner>

    <button mat-icon-button matSuffix *ngIf="exampleField.value && !isLoading" (click)="exampleField.reset()">
      <mat-icon>close</mat-icon>
    </button>

    <mat-error *ngIf="exampleField?.hasError('customErrorName')">
      Invalid Example Field value
    </mat-error>
  </mat-form-field>
</form>
```

File: src/app/pet-form/pet-form.component.html [hunk 2]
```diff
@@ -1,108 +1,105 @@
...
+          accept="image/*">
+      </p>
+    </div>
     <h3 class="inline">Prescriptions</h3>
     <button mat-mini-fab color="primary" aria-label="icon button with a plus sign to add new prescription"
       class="inline" (click)="addPrescription()" type="button">
       <mat-icon>add_icon</mat-icon>
     </button>
-    <div formArrayName="prescriptions">
+    <div formArrayName="prescriptions" class="border-bottom">
```
Comment (swhook52) [inline src/app/pet-form/pet-form.component.html]: Remember to name classes based on the functions they serve and not the styles they represent. For example you might have a "card" style or a "prescription-card" but wouldn't have a "add-margin" or "extra-space" class.

**Comment by swhook52** [comment]
Great work on this stuff, especially the dialog use as that's exactly how you use them. I can merge this whenever you're ready but I wasn't sure if you wanted to wait for the last PR to come through first?

File: src/app/pet-form/pet-form.component.html (hunk 2 above)
Comment (emcapone) [reply src/app/pet-form/pet-form.component.html]: I believe that I corrected this issue in the new commit. If you still see room for improvement, please do let me know.

File: src/app/account-profile/account-profile.component.html (hunk 1 above)
Comment (emcapone) [reply src/app/account-profile/account-profile.component.html:2]: I will look into Reactive Forms. Thank you!

**Pull request merged**